tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{27,36}-pylint{17,18,19}, py{36,37}-pylint{20,21,22}, coverage, pylint
+envlist = py{35,36,37}-pylint{17,18,19,20,21,22,23,24}, coverage, pylint
```

Comment:
Do we really need to keep testing against this many pylint versions?

Reply:
Killed most of the old versions.